<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Validation</class>
 <widget class="QDialog" name="Validation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Кросс-валидация нейронной сети</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,0">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_start_validation">
        <property name="text">
         <string>Запустить</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_groupAmount">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>83</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Кол-во групп</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_groupAmount">
          <property name="minimumSize">
           <size>
            <width>33</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>33</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop_validation">
        <property name="text">
         <string>Остановить </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Прогресс выполнения</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	background-color: white;
	border: 1.5px solid black;
}

QProgressBar::chunk {
	background-color: rgb(255,192,203);
    width: 15px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="label_stage_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_stage_number">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
